### ISSUE TITLE
Data preservation?

### ISSUE THREAD

**Issue by pnill** [BlazingTwist/SoD_Modding#1]
Curious if you have any plans to preserve data as it currently exists, including the MMO side packets using your other project?

Someone had posted about this on reddit today, and it always gets me curious to look into something when it's being shutdown briefly analyzing this and realizing it's Unity based kind of indicated it'd be a simple feat to potentially emulate but after observing playfab and other APIs as well as what seems to be online only content being downloaded in real time every play... seems like it'd be a bit difficult. Then realized you have a downloader included into this.

So basically just asking if you personally are working to preserve any of the online only stuff, including any packet data which wouldn't be easily replicated (quets/dialog info, etc.) which all seems to be downloaded from their servers/cdn.

**Comment by BlazingTwist** [comment]
Personally I simply don't have the time to properly archive the server data on my own.
I tried for about 2 or 3 months, 2 years ago when I wasn't working full time yet. https://github.com/BlazingTwist/SoD_OfflineServer
But "quickly" noticed that there are too many server-side unknowns, at least for my skill level at the time.

So the project has kind of become something that I'm "scared" to work on.

Though I don't fully want to drop the project either.
I guess a good start would be to gather the data that's easily accessible.
With the servers shutting down soon, at least that means that if I create the database again, it won't be outdated a few months after.
